package.json

```diff
@@ -10,7 +10,7 @@
     "build": "npx tsc",
     "prestart": "npm run build",
     "start": "node dist/server.js",
-    "dev": "NODE_ENV=development tsc-watch --onSuccess \"node ./dist/server.js\"",
+    "dev": "tsc-watch --onSuccess \"node ./dist/server.js\"",
```

Comment:
Why have we removed this?

Reply:
This was causing error on my windows machine and hence is temporarily removed just for this draft pr.

Reply:
We will need to use the cross-env lib for this. Can you add it?